src/helpers/getCourseById.ts

```diff
@@ -0,0 +1,10 @@
+import { mockedCoursesList as data } from 'src/constants';
+
+export const getCourseById = (id: string): unknown => {
```

Comment:
`.find` can be used here

```suggestion
export const getCourseById = (id: string): unknown => {
  return mockedCoursesList.find((course) => course.id === id)
}
```